src/dex-ui-components/base/Button/SettingsButton.tsx

```diff
@@ -1,21 +1,24 @@
 import { Text } from "@chakra-ui/react";
-import { isNil } from "ramda";
+import { Color } from "../..";
+import { DefaultPercent } from "../../../dex-ui/utils";
 import { SettingsIcon } from "../Icons";
 import { Button } from "./Button";
 
 interface SettingsButtonProps {
+  isError?: boolean;
   slippage: number;
   onClick?: React.MouseEventHandler<HTMLButtonElement>;
 }
 
 export function SettingsButton(props: SettingsButtonProps) {
-  const formattedSlippage = !isNil(props.slippage) ? `${Number(props.slippage)?.toFixed(1)}%` : "-.-%";
+  const formattedSlippage = props.slippage > 0 ? `${Number(props.slippage)?.toFixed(2)}%` : DefaultPercent;
```

Comment:
Rounding to a single decimal produced a poor experience in the defi forms. For example, a user input slippage of `0.25` would round to `0.3` in the settings button. Now an input of `0.25` will round to `0.25` - this is much easier to check against the price impact if the settings inputs are hidden.